json-endpoints/class.wpcom-json-api-get-site-endpoint.php

```diff
@@ -298,6 +298,13 @@ public function build_current_site_response( ) {
 
 				if ( $is_jetpack ) {
 					$response['options']['jetpack_version'] = get_option( 'jetpack_version' );
+					$response['options']['json_api_full_management'] = get_option( 'jetpack_json_api_full_management', false );
```

Comment:
The line above is out of date since we don't use 'jetpack_json_api_full_management' any more.